app/models/journey.rb

```diff
@@ -46,11 +46,16 @@ def keywords_search
     join_search([to_search_str(keywords), steps_search(:keywords)])
   end
 
+  def new_step
+    new_position = steps.empty? ? 1 : steps.last.position + 1
+    steps.new(position: new_position)
+  end
+
   def reposition
     steps.each_with_index do |step, index|
-      position_by_index = (index + 1) * 2
+      position_by_index = index + 1
       if step.position != position_by_index
```

Comment:
Tu myslim staci `step.position = index + 1` a potom `position.save` to urobi update ak treba.